app/jobs/irs_attempts_events_batch_job.rb

```diff
@@ -3,8 +3,7 @@ class IrsAttemptsEventsBatchJob < ApplicationJob
 
   def perform(timestamp = Time.zone.now - 1.hour)
     enabled = IdentityConfig.store.irs_attempt_api_enabled &&
-              IdentityConfig.store.irs_attempt_api_aws_s3_enabled &&
-              IdentityConfig.store.irs_attempt_api_bucket_name
+              s3_helper.attempts_bucket_enabled
     return nil unless enabled
```

Comment:
I suppose we could inline this whole thing now to just
`return nil unless IdentityConfig.store.irs_attempt_api_enabled && s3_helper.attempts_bucket_enabled` if that fits on one line.

Reply:
It does!  Making that change